app/routes/search_label.js

```diff
@@ -2,6 +2,14 @@ import AuthorizeRoute from './authorize';
 
 export default AuthorizeRoute.extend({
 
+  afterModel(model, transition){
+    var pkgs = model.get("packages");
+    if(pkgs && pkgs.length > 0 && (pkgs.get("firstObject.hasAllPackagesDesignated") || pkgs.get("firstObject.hasAllPackagesDispatched"))){
+      transition.abort();
+      this.get('messageBox').alert(this.get("i18n").t('designated_dispatched_error'));
```

Comment:
After aborting the transition and pressing okay it will only show blue page? Do we need to redirect user at click of okay button?